### PR TITLE
ENGDOCS-2141

### PR DESCRIPTION
**Pull request by aevesdocker**
## Description

Adds some examples as per Learning path discussion and adjusts some header levels to acommodate  the examples section

## Reviews

- [x] Technical review
- [ ] Editorial review
- [x] Product review

### CODE REVIEW

File: content/compose/compose-file/05-services.md [hunk 1]
```diff
@@ -20,7 +20,58 @@ available resources. Deploy support is an optional aspect of the Compose Specifi
...
+  web:
+    image: nginx:latest
+    ports:
+      - "8080:80"
+
+  db:
+    image: postgres:13
+    environment:
+      POSTGRES_USER: example
+      POSTGRES_PASSWORD: example
```
Comment (glours) [inline content/compose/compose-file/05-services.md:38]: Not sure we should show this kind of setup, users will reproduce it and it's definitely not a good practice, we should use a secret for this or at least setup the password value via an env variable. 
for example
```suggestion
      POSTGRES_PASSWORD: ${POSTGRES_USER_PASSWORD}
```
Or just remove the `POSTGRES_PASSWORD` definition in this example

**Review by glours — APPROVED** [review]
Just update the first sample to avoid hard coded password and it's good to go

File: content/compose/compose-file/05-services.md [hunk 2]
```diff
@@ -20,7 +20,57 @@ available resources. Deploy support is an optional aspect of the Compose Specifi
...
+      - "8080:80"
+
+  db:
+    image: postgres:13
+    environment:
+      POSTGRES_USER: example
+      POSTGRES_DB: exampledb
+```
+
+This minimal example demonstrates how to define two simple services, set their images, map ports, and configure basic environment variables using Docker Compose.
```
Comment (dvdksn) [inline content/compose/compose-file/05-services.md:41]: I suggest that we lead with the descriptions instead

File: content/compose/compose-file/05-services.md [hunk 3]
```diff
@@ -20,7 +20,57 @@ available resources. Deploy support is an optional aspect of the Compose Specifi
...
+
+  backend:
+    build:
+      context: backend
+      target: builder
+```
+
+The `proxy` service uses the Nginx image, mounts a local Nginx configuration file into the container, exposes port `80` and depends on the `backend` service. 
+
+The `backend` service builds an image from the Dockerfile located in the `backend` directory that is set to build at stage `builder`.
```
Comment (dvdksn) [inline content/compose/compose-file/05-services.md:67]: The same here, I would put the description before, to introduce the example.

**Review by dvdksn — APPROVED** [review]

**Pull request merged**